python/ray/includes/unique_ids.pxi

```diff
@@ -212,15 +212,56 @@ cdef class ClientID(UniqueID):
         return <CClientID>self.data
 
 
-cdef class JobID(UniqueID):
+cdef class JobID(BaseID):
+    cdef CJobID data
 
+    # TODO(qwang): Should we remove this ctor?
```

Comment:
1) `ctor` could be confusing, please spell it out.
2) I don't think we can remove it? How do you construct a JobID then?